doc/firehol/introduction.md

```diff
@@ -46,7 +46,11 @@ To build and install taking the default options:
     ./configure && make && sudo make install
           
 
-Alternatively, just copy the `sbin/firehol.in` file to where you want it.
+Since all components (including configuration files) will go
+under `/usr/local`, you may prefer to configure more like this:
+
+    ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var --libexecdir=/usr/lib
```

Comment:
This is just docs; I just wanted an easy reference for the most commonly used options. Generally, you are right the packager will see these and an end user won't.